README.md

```diff
@@ -94,13 +94,91 @@ export ETCDCTL_ENDPOINT=https://{ip of etcd host}:2379
 etcdctl member list
 ```
 
+# Operational Actions
+
+### Restore
+
+Allows the operator to restore the data from a cluster-data snapshot. This
+comes with caveats and a very specific path to restore a cluster:
+
+The cluster must be in a state of only having a single member. So it's best to
+deploy a new cluster using the etcd charm, without adding any additional units.
+
+```
+juju deploy etcd new-etcd
+```
+
+> The above code snippet will deploy a single unit of etcd, as 'new-etcd'
+
+Once the restore action has completed, evaluate the cluster health. If the unit
+is healthy, you may resume scaling the application to meet your needs.
```

Comment:
The first time I read this, I missed the "restore action" bit and thought it was another comment on the `juju deploy etcd new-etcd` command. Oops.

Should this be moved after the `juju run-action ...` example below? I think it might flow a little better that way.